### PR TITLE
[3.7.2] Release materials

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -153,6 +153,34 @@ Multiple performance enhancements for Distributed server mailbox, IMAP, SMTP and
...
  - JAMES-3433 MimeMessageStore StoragePolicy should be the same on read and writes
 
+## [3.7.2] - 2022-10-06
+
+### Security 
+
+ - [UPGRADE] scala-library 2.13.7 -> 2.13.9
+ - [CONF] Remove vendor URLs from smtpserver.xml default configuration examples (#1205)
+ - [Upgrade] Maven slf4j 1.7.32 -> 2.0.1
+ - [Upgrade] Maven netty4 4.1.72.Final -> 4.1.81.Final
```
Comment (Arsnael) [inline CHANGELOG.md:163]: I looked a bit closer at this one and I'm a bit confused...

It refers to pretty much this commit: https://github.com/apache/james-project/commit/24b87c8bd3a320e0f01e6c2010067ff808688513

In 3.7.x we seem to have netty4 on blob-s3 and queue-activemq modules... while if I look in the pom on the root of the project, we are still at the version 3 -> 3.10.6.Final

Isn't that a problem potentially?

Comment (chibenwa) [reply CHANGELOG.md:163]: No, those are on different packages -> no impact

**Review by Arsnael — APPROVED** [review]

File: src/site/xdoc/server/release-notes.xml [hunk 2]
```diff
@@ -39,6 +39,10 @@
                 of a <a href="https://github.com/apache/james-project/blob/master/CHANGELOG.md">changelog file</a>
                 as well as details <a href="https://github.com/apache/james-project/blob/master/upgrade-instructions.md">upgrade instructions</a>.</p>
         </section>
+        <section name="Version 3.7.2">
+            <p>Released March 2021</p>
```
Comment (quantranhong1999) [inline src/site/xdoc/server/release-notes.xml:43]:
```suggestion
            <p>Released October 2022</p>
```

File: src/site/xdoc/server/release-notes.xml [hunk 3]
```diff
@@ -39,6 +39,10 @@
...
+            <p><a href="https://james.apache.org/james/update/2022/10/06/james-3.7.2.html">Read the release announce</a>. </p>
+        </section>
         <section name="Version 3.7.1">
             <p>Released March 2021</p>
```
Comment (quantranhong1999) [inline src/site/xdoc/server/release-notes.xml:47]:
```suggestion
            <p>Released August 2022</p>
```

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**